example_13/doc/userdoc.rst

```diff
@@ -0,0 +1,474 @@
+    rrbot_with_sensor_fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster] active
+    rrbot_with_sensor_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    rrbot_with_sensor_position_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    threedofbot_pid_gain_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_position_controller[forward_command_controller/ForwardCommandController] active
+
+
+6. Deactivate ``RRBotSystemPositionOnly`` and its position controller (first). Call
+
+  .. code-block:: shell
+
+    ros2 control switch_controllers --deactivate rrbot_position_controller
+    ros2 service call /controller_manager/set_hardware_component_state controller_manager_msgs/srv/SetHardwareComponentState "
+    name: RRBotSystemPositionOnly
+    target_state:
+      id: 0
+      label: inactive"
+
+  Scenario state:
+
+  - right robot is now "standing" at the last position
+  - left robot is moving
+  - middle robot is moving
+
+  Hardware status: ``RRBotSystemPositionOnly`` is in inactive state.
+
+  .. code-block:: shell
+
+    $ ros2 control list_hardware_components
+    ...
+    Hardware Component 4
+        name: RRBotSystemPositionOnly
+        type: system
+        plugin name: ros2_control_demo_example_5/RRBotSystemPositionOnlyHardware
+        state: id=2 label=inactive
+        command interfaces
+                rrbot_joint1/position [available] [unclaimed]
+                rrbot_joint2/position [available] [unclaimed]
+    ...
+
+  Controllers status: ``rrbot_position_controller`` is now in inactive state
+
+  .. code-block:: shell
+
+    $ ros2 control list_controllers
+    joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    rrbot_external_fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster] active
+    rrbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    rrbot_position_controller[forward_command_controller/ForwardCommandController] inactive
+    rrbot_with_sensor_fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster] active
+    rrbot_with_sensor_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    rrbot_with_sensor_position_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    threedofbot_pid_gain_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_position_controller[forward_command_controller/ForwardCommandController] active
+
+
+7. Set ``RRBotSystemPositionOnly`` in unconfigured state, and deactivate its joint state broadcaster. Also restart global joint state broadcaster. Call
+
```

Comment:
With the 7th point, I don't get the same result. For me all the robots stopped moving and the following is the output from `list_controllers` command

```
$ ros2 control list_controllers
rrbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] inactive  
threedofbot_pid_gain_controller[forward_command_controller/ForwardCommandController] active    
threedofbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active    
rrbot_with_sensor_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active    
joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] inactive  
rrbot_with_sensor_fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster] active    
rrbot_with_sensor_position_controller[forward_command_controller/ForwardCommandController] active    
rrbot_position_controller[forward_command_controller/ForwardCommandController] inactive  
rrbot_external_fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster] active    
threedofbot_position_controller[forward_command_controller/ForwardCommandController] active

Reply:
That's true. The joint_state_broadcaster cannot be activated again (the cli returns a success, but it is not activated).
Because of the verbose output described with https://github.com/ros-controls/ros2_control/issues/1103 I was not able to see any error message from the controller_manager, I don't know how to debug this now.

Reply:
Thanks for getting back. If I get some time, I will try to take a look at this issue.
Thank you for reporting it

Reply:
This seems to be fixed in the meantime?

Reply:
Solved!